tests/scripts/mbed-build.sh

```diff
@@ -0,0 +1,107 @@
+#! /usr/bin/env sh
+
+# mbed-build.sh
+#
+# This file is part of Mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2019, Arm Limited, All Rights Reserved
+#
+# Purpose
+#
+# To run test builds of the Mbed OS module for all specified targets.
+# Usage: mbed-build <folder to import to> <optinally string of space seperated targets>
```

Comment:
Minor: Reading the code, it looks like "space separated" means: `./program.sh "A B C"` instead of `./program.sh A B C`. This is only a minor comment, but note that "space separated" in this kind of way is perhaps not the best choice because thats what the shell uses to distinguish arguments to the programs. I suggest to instead modify the program to use either separate arguments for a variable number of targets OR use a different separator (for example `,`).